docs/user-guide/scheduled-jobs.md

```diff
@@ -19,6 +19,9 @@ A _Scheduled Job_ manages time based [Jobs](/docs/user-guide/jobs/), namely:
 One ScheduledJob object is like one line of a _crontab_ (cron table) file. It runs a job periodically
 on a given schedule, written in [Cron](https://en.wikipedia.org/wiki/Cron) format.
 
+NOTE: The question mark (`?`) in the schedule has the same meaning as an asterisk `*`,
+that is, it stands for any of available value for a given field.
```

Comment:
No particular reason, I guess I used to write it in that since I remember, that's why I'm leaving this as is. Even more, this shows of other possibilities. So I'm not strongly in favor of either. If  you want me to, I can change that.